homeassistant/components/rako/config_flow.py

```diff
@@ -0,0 +1,78 @@
+"""Config flow for Rako."""
+import asyncio
+from typing import Any, Dict, Optional
+
+from python_rako import Bridge, discover_bridge
+from python_rako.const import RAKO_BRIDGE_DEFAULT_PORT
+from python_rako.exceptions import RakoBridgeError
+from python_rako.model import BridgeInfo
+import voluptuous as vol
+
+from homeassistant.components.rako.const import CONF_MAC_ADDRESS, DOMAIN
```

Comment:
This is a known issue, you can disable this check on this specific point in the code using 

`# pylint:disable=unused-import`

Also, please use relative import paths for imports from the same package:

`from homeassistant.components.rako.const` -> `from .const`